### ISSUE TITLE
WCAG: 3.2 Predictable

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#4134]
### Requirement
> Make Web pages appear and operate in predictable ways.
https://www.w3.org/TR/WCAG22/#predictable

#### :x: on focus
On focus acts appropriately by not triggering a change of context.  However in terms of links, we have not consistently differentiated between internal page links (there are a lot of cross references in articles), external links opening in the same window, and external links which open in new tabs.  Internal page links are programatically determined such that screen readers do identify them correctly as in-page links.  

#### :x: on input
When presented with a select from which to choose, a user might rightly predict there would be at least two options.  We present the user with a language select input, even when there is only one language, so no alternative selection is possible.

#### ✅ consistent navigation
The same navigation is use across all pages, so it is identical on each page.

#### ✅ consistent identification
By design, components with the same functionality are repeats of the same template with, for example, a different endnote, and thus their format consistent.

#### ✅ consistent help
Help is via contact form, which is found in the navigation rather than appearing on multiple pages.

### Work to do
- #4221 
- #4248
